Helpers/NCHelper.m

```diff
@@ -0,0 +1,35 @@
+//
+//  NCHelper.m
+//  social-maps
+//
+//  Created by César Francisco Barraza on 7/23/18.
+//  Copyright © 2018 Bevin Benson. All rights reserved.
+//
+
+#import "NCHelper.h"
+
+@implementation NCHelper
++ (void)addObserver:(nonnull id)observer type:(NotificationType)type selector:(SEL)selector {
+    NSString* notification = [NCHelper stringWithNotificationType:type];
+    [[NSNotificationCenter defaultCenter] addObserver:observer selector:selector name:notification object:nil];
+}
+
++ (void)notify:(NotificationType)type object:(nullable id)object {
+    NSString* notification = [NCHelper stringWithNotificationType:type];
+    [[NSNotificationCenter defaultCenter] postNotificationName:notification object:object];
+}
+
++ (NSString*)stringWithNotificationType:(NotificationType)type {
+    switch(type)
+    {
+        case NTAddFavorite:
+            return @"AddFavoriteNotification";
+        case NTAddToWishlist:
+            return @"AddToWishlistNotification";
+        case NTNewFollow:
+            return @"NewFollowNotification";
+    }
+    
+    return @"";
```

Comment:
One concern with doing something like this is that, if someone forgets to enumerate a case (I'm not sure if the compiler catches this or not), you'll return @"" without knowing it. 

You could throw in a nslog or an assert to watch for that: https://developer.apple.com/documentation/foundation/nsassert?language=objc

Reply:
Yeah, we could make it throw an exception if something like that happened; the main purpose of this is to avoid similar problems while using the default one, after all.

Reply:
Thanks!